### PR TITLE
fix up readme on building with MSVC (Visual Studio)

### PR DESCRIPTION
**Pull request by BlaiZephyr**
re-added and updated building with Visual Studio.

[reason](https://github.com/ddnet/ddnet/pull/8195#issuecomment-2088560288)

## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Review by heinrich5991 — COMMENTED** [review]
Thanks for the quick PR!

File: README.md [hunk 1]
```diff
@@ -196,7 +196,20 @@ Check the SAN.\* files afterwards. This finds more problems than memcheck, runs
 For valgrind's memcheck compile a normal Debug build and run with: `valgrind --tool=memcheck ./DDNet`
 Expect a large slow down.
 
-Building on Windows with Microsoft Visual C++
+Building on Windows with the Visual Studio IDE
+--------------------------------------
+
+Download and install some version of [Microsoft Visual Studio](https://www.visualstudio.com/) (as of writing, MSVS Community 2022) with **C++ support**.
```
Comment (murpii) [inline README.md:202]: At the time of writing / As of the time of writing

File: README.md [hunk 2]
```diff
@@ -196,7 +196,20 @@ Check the SAN.\* files afterwards. This finds more problems than memcheck, runs
...
+
+You'll have to install both [Python 3](https://www.python.org/downloads/) and [Rust](https://rustup.rs/) aswell.
```
Comment (murpii) [inline README.md:204]: as well

File: README.md [hunk 3]
```diff
@@ -196,7 +196,20 @@ Check the SAN.\* files afterwards. This finds more problems than memcheck, runs
...
+
+Make sure the MSVC build tools, C++ CMake-Tools and the latest Windows SDK version appropriate to your windows version are selected in the installer.
+
+Now open up your Project folder, Visual Studio should automatically detect and configure your project using cmake.
```
Comment (murpii) [inline README.md:208]: CMake, for consistency

**Comment by BlaiZephyr** [comment]
fixed up and done - thanks for pointing out my horrible grammar @murpii haha

**Pull request merged**